packages/request-client.js/src/api/request.ts

```diff
@@ -108,6 +128,9 @@ export default class Request {
     this.skipPaymentDetection = options?.skipPaymentDetection || false;
     this.disableEvents = options?.disableEvents || false;
     this.currencyManager = currencyManager;
+    this.topics = options?.topics;
+    this.transactionData = options?.transactionData;
+    this.paymentRequest = options?.paymentRequest;
```

Comment:
Does it make sense to have 1 or 2 of these objects defined? Shouldn't they all be either defined or undefined?

Reply:
Merged into a single object that is either defined or undefined. :white_check_mark: